frontend/src/components/personal/preferencePage/PreferencesPageAvatar.tsx

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable react/jsx-no-bind */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+
+import React, { useState, useCallback } from 'react'
+import { Avatar, Button, Typography } from '@material-ui/core'
+import Modal from '@material-ui/core/Modal'
+import { useFormik } from 'formik'
+import {useDispatch, useSelector } from 'react-redux'
+
+import { useStyles, useStyleModal } from '../profilePage/ProfilePreferencesPage.styles'
+import {validationSchema} from '../../../utils/ValidationSchemes'
+import {updateUserAvatar,
+  userPreferencesRequestFailed} from '../../../actions/prefAndProfileActions'
+import {RootState} from '../../../reducers/index'
+  
+export const PreferencesPageAvatar = (): React.ReactElement => {
+    const avatar = useSelector((state: RootState) => state.operatePreferencesDataReducer.avatar)  
+    const dispatch = useDispatch() 
+    
+    const [enter, setEnter] = useState(false)
+    const [open, setOpen] = useState(false)
+  
+    const mouseEnter = useCallback(() => setEnter(true),[])
+    const mouseLeave = useCallback(() => setEnter(false),[])
+
+    const classes = useStyles()
+    const modalClass = useStyleModal()
+    
+    const handleOpen = useCallback(() => setOpen(true), [setOpen])
+    const handleClose = useCallback(() => setOpen(false), [setOpen])
+
+    const updateAvatar = useCallback(() => {
+        handleOpen()
+    },[handleOpen])
+
+    const formik = useFormik({
+      enableReinitialize: true,
+      initialValues: {
+        file: '',
+      },
+      validationSchema: validationSchema.validationFile,
+      onSubmit: (values) => {
+        console.log(values)
+        const data = new FormData()
+        if (values.file !== null) {
+          data.append('avatar', values.file)
+        }
+        try {
+          //@ts-ignore
```

Comment:
Will this `catch` work? Did you test it? For instance: you can add `console.log` or `alert` to make sure that exactly _this_ `catch` is triggered.

Reply:
Seems that we dont have any errors that can be caught on this try-catch block - errors from Form are caught and shown on validation, HttpClient errors on Api request are Thrown to action` try catch and there they go to be shown to user. Removed try-catch from there.